docs/Installation/server-installation.md

```diff
@@ -9,9 +9,11 @@ To install PaperTrail you will first need to install the prerequisites:
 
 Once you have the prerequisites : 
 
-1.  Download and run the installer - this can be done silently using the **-q** option  
```

Comment:
See https://github.com/egis/awesome#markdown use semantic markdown e.g. `PaperTrail`.
The default convention is also to use `papertrail`